In [15]:
name = "Erica"
hourly_wage = 22.75
hours = 40
weekly_pay = hours * hourly_wage
print(f"{name} earns {weekly_pay} per week")

Erica earns 910.0 per week


In [16]:
temps = [61, 65, 72, 58, 70, 74, 68]
days_above_65 = 0
for t in temps:
  if t >=65:
    days_above_65 = days_above_65 + 1
print(days_above_65)

5


In [17]:
def to_celsius(f):
  return (f - 32) * 5/9

print(round(to_celsius(temps[0]), 1))

16.1


In [18]:
movie = {"title": "Iron Lung", "year": 2026, "rating": 5.8}
for key, value in movie.items():
  print(key, ":", value)

title : Iron Lung
year : 2026
rating : 5.8


In [19]:
import pandas as pd
df = pd.read_csv("chickwts.csv")
df.head()

,Unnamed: 0,weight,feed
0,1,179,horsebean
1,2,160,horsebean
2,3,136,horsebean
3,4,227,horsebean
4,5,217,horsebean


In [20]:
df.describe()

,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


This data shows  there are 71 instances of chicken weight documented in the file along with various spreads to analyze the data. We know that the average weight is around 261.3 pounds with deviations of about 78.1 pounds between each chicken, along with the fact the heavest chicken is 423 pounds while the lightest chicken is 108 pounds. We can also see the weight of the chicken broken down into precentiles, where chicken weighing around 204.5 pounds are in the 25th percentile, chickens weighing around 258 pounds are in the 50th, and chickens weighing around 323.5 pounds are in the 75th percentile.